adafruit_mcp9808.py

```diff
@@ -109,9 +109,81 @@ def temperature(self):
         with self.i2c_device as i2c:
             i2c.write_then_readinto(self.buf, self.buf, out_end=1, in_start=1)
 
+        return self._temp_conv()
+
+    def _temp_conv(self):
         # Clear flags from the value
         self.buf[1] = self.buf[1] & 0x1F
         if self.buf[1] & 0x10 == 0x10:
             self.buf[1] = self.buf[1] & 0x0F
             return (self.buf[1] * 16 + self.buf[2] / 16.0) - 256
         return self.buf[1] * 16 + self.buf[2] / 16.0
+
+    def _limit_temperatures(self, temp, t_address=0x02):
+        """Internal function to setup limit temperature
+
+        :param int temp: temperature limit
+        :param int t_address: register address for the temperature limit
+                                0x02 : Upper Limit
+                                0x03 : Lower Limit
+                                0x04 : Critical Limit
+        """
+
+        if temp < 0:
+            negative = True
+            temp = abs(temp)
+        else:
+            negative = False
+
+        self.buf[0] = t_address
+
+        self.buf[1] = temp >> 4
+        if negative:
+            self.buf[1] = self.buf[1] | 0x10
+
+        self.buf[2] = (temp & 0x0F) << 4
+
+        with self.i2c_device as i2c:
+            i2c.write(self.buf)
+
+    @property
+    def upper_temperature(self):
+        """Upper temperature property"""
```

Comment:
Document properties like you would a variable rather than a function because they look like variables from the outside. For example, this could be "Upper alarm temperature in Celsius". You don't need a comment for the setter. The action of getting or setting is implicit in how you use it.

Reply:
Solved in https://github.com/adafruit/Adafruit_CircuitPython_MCP9808/pull/29/commits/28ac104d2b0428c938cfa8da6ca0f43610a786eb